libs/core/src/lib/combobox/auto-complete.directive.ts

```diff
@@ -123,4 +126,9 @@ export class AutoCompleteDirective {
             forceClose: forceClose
         });
     }
+
+    /** @hidden */
+    private _moveIndicatorToLastLetter(): void {
```

Comment:
I think the name of the method does not match what the function is actually doing. It's moving the cursor after the last character not "to the last letter".